resources/views/match/createMatch.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('layouts.plain')
+@section('title','Create Match')
+
+@section('content')
+    <div class="container" id="app">
```

Comment:
you need to add a place to show the form errors somewhere here - for the validation I mentioned before.